homeassistant/components/wiz/translations/de.json

```diff
@@ -8,7 +8,8 @@
             "bulb_time_out": "Es kann keine Verbindung zur Gl\u00fchbirne hergestellt werden. Vielleicht ist die Gl\u00fchbirne offline oder es wurde eine falsche IP/Host eingegeben. Bitte schalte das Licht ein und versuche es erneut!",
             "cannot_connect": "Verbindung fehlgeschlagen",
             "no_wiz_light": "Die Gl\u00fchbirne kann nicht \u00fcber die Integration der WiZ-Plattform verbunden werden.",
-            "unknown": "Unerwarteter Fehler"
+            "unknown": "Unerwarteter Fehler",
+            "no_ip": "Es wurde keine g\u00fcltige IP-Adresse eingegeben."
```

Comment:
This doesn't matter. The source of truth for translations is in Lokalise and this file will be overwritten

Reply:
Removed